packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -366,6 +366,22 @@ export class Autosuggest {
     return `${this.inputId}-${this.suggestions.indexOf(suggestion) + 1}`;
   }
 
+  private getChunkedExtras(extras: string[]): string[][] {
+    if (!extras?.length) {
+      return [];
+    }
+
```

Comment:
Wat mij betreft is deze code overbodig: De signature van deze method dwingt af dat er een array is, `extras?` voegt niets toe.

Daarnaast, een reduce over een lege array geeft meteen de initial value terug, wat ook een lege array is 🤷

Reply:
✅